Exercise_6/Exercise_6.ipynb

```diff
@@ -219,7 +255,8 @@
     "\n",
     "1. Find the list of commits and examine the files for a specific commit.  \n",
     "\n",
-    "2. Modify a file and use the web interface to make a new commit.  \n",
+    "2. Modify a file and use the web interface to make a new commit. \n",
+    "![Edit_file_GitHub.png](attachment:Edit_file_GitHub.png)\n",
```

Comment:
I think the example image should actually show the git-example repo not the git-course repo?

Reply:
But we could also just use the git-course repo instead?
No need to add a second repo here.
What do you think?